BossMod/Components/BaitAway.cs

```diff
@@ -178,22 +178,45 @@ public override void OnUntethered(BossModule module, Actor source, ActorTetherIn
     public class BaitAwayCast : GenericBaitAway
     {
         public AOEShape Shape;
+        public bool ResolvesAtCastEvent;
+        public bool IsCharge;
+        public float HalfWidthCharge;
```

Comment:
This should be a separate component (eg without Shape field). Otherwise - what's the purpose of defining a Shape for a charge? The constructor would still require it, etc.

Reply:
ok, i will try to rework it later into a seperate component